expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -724,6 +726,39 @@ void shouldReturnFundingSourceAsObjectWhenFundingSourceLacksLabels() throws Json
         assertFalse(expandedResource.at(JsonPointer.compile("/fundings/0/identifier")).isMissingNode());
     }
 
+    @Test
+    void shouldReturnFundingSourceAsObjectWhenLackingIdentifier() throws JsonProcessingException,
+                                                                         BadRequestException {
+        var publication = randomPublication();
+        var source = URI.create("https://api.test.nva.aws.unit.no/cristin/funding-sources/OTHER");
+        var unconfirmedFunding = (UnconfirmedFunding) new FundingBuilder()
+                                                          .withSource(source)
+                                                          .withFundingAmount(randomMonetaryAmount())
+                                                          .withLabels(randomLabels())
+                                                          // no identifier
+                                                          .build();
```

Comment:
Not a fan of comments, we have well-named methods.
```suggestion
        var unconfirmedFunding = createUnconfirmedFundingWithNoIdentifier();
```

Reply:
Its hard to see whats not there (withIdentifier is removed), so then I like a comment to point it out. Not sure how to do FundingBuilder